symfony/src/Dto/CustomerDTO.php

```diff
@@ -43,7 +39,6 @@ final class CustomerDTO
     /**
      * @Assert\Type("string")
      * @Assert\Length(min = 8, max = 20, minMessage = "minimal length is 8 numbers in a string", maxMessage = "maximal length is 20 numbers in a string")
```

Comment:
nit: message can be omitted as the default message contains all necessary description and also translated into several languages.